docs/best_practice_guide.md

```diff
@@ -0,0 +1,127 @@
+# Best practice guide
+
+## What to use for
+
+You can use typo3-rector in various ways:
+- checking existing code if there are left out segments of the last upgrade
+- reformating e.g. typoscript files based on your editorconfig
```

Comment:
No. Rector is not reformatting all files but only changed ones

Reply:
ok, will change